tests/test_index.py

```diff
@@ -353,7 +353,7 @@ def test_index_noarch_osx64_1(testing_workdir):
                 "home": "https://anaconda.org/conda-test/conda-index-pkg-a",
```

Comment:
can we delete this entire file, as a nearly identical one exists in conda-index?

Reply:
Yes we can